node/resolvers/catalog/assemblyOption.ts

```diff
@@ -46,11 +53,18 @@ function defineInputValueType(inputValue: RawInputValue): InputValueType {
   return InputValueType.TEXT
 }
 
+function stringToBoolean(value: string) {
+  return value === 'true'
```

Comment:
I could be _smart_ and just do this instead of the function, but I think this makes the code clearer to read.